cli/src/main/java/com/coderanch/blackjack/InputUtilities.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2018 Coderanch.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.coderanch.blackjack;

```

Comment:
You mean all important .java classes of internal parts of the model will be in com.coderanch.blackjack package? 
will it be okay if i create package com.coderanch.ui where UserInterface class will reside ?

Reply:
Would that package name make sense considering that CodeRanch might write UIs for different applications? Or that it might even write multiple different UIs for Blackjack?

I propose `com.coderanch.blackjack.cli`.

Reply:
Ah! I see, that way. Got now.